project/AppDependencies.scala

```diff
@@ -5,31 +5,35 @@ import sbt._
 
 object AppDependencies {
 
+  val playVersion = "play-28"
+
   val compile = Seq(
-    "uk.gov.hmrc"                %% "govuk-template"             % "5.66.0-play-26",
-    "uk.gov.hmrc"                %% "play-ui"                    % "9.4.0-play-26",
-    "uk.gov.hmrc"                %% "bootstrap-frontend-play-26" % "5.3.0",
-    "uk.gov.hmrc"                %% "mongo-caching"              % "7.0.0-play-26",
-    "uk.gov.hmrc"                %% "domain"                     % "5.11.0-play-26",
-    "uk.gov.hmrc"                %% "play-language"              % "5.0.0-play-26",
-    "org.typelevel"              %% "cats-core"                  % "2.1.0",
-    "com.github.kxbmap"          %% "configs"                    % "0.4.4",
-    "org.julienrf"               %% "play-json-derived-codecs"   % "7.0.0",
-    "com.github.julien-truffaut" %% "monocle-core"               % "2.0.0",
-    "com.github.julien-truffaut" %% "monocle-macro"              % "2.0.0",
-    compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full),
-    "com.github.ghik" % "silencer-lib" % "1.7.1" % Provided cross CrossVersion.full
+    "uk.gov.hmrc"                %% "govuk-template"                    % s"5.68.0-$playVersion",
+    "uk.gov.hmrc"                %% "play-ui"                           % s"9.4.0-$playVersion",
+    "uk.gov.hmrc"                %% s"bootstrap-frontend-$playVersion"  % "5.12.0",
+    "uk.gov.hmrc"                %% "mongo-caching"                     % s"7.0.0-$playVersion",
+    "uk.gov.hmrc"                %% "domain"                            % s"6.1.0-$playVersion",
+    "uk.gov.hmrc"                %% "play-language"                     % s"5.0.0-$playVersion",
+    "org.typelevel"              %% "cats-core"                         % "2.6.0",
+    "com.github.kxbmap"          %% "configs"                           % "0.6.1",
+    "org.julienrf"               %% "play-json-derived-codecs"          % "10.0.2",
+    "com.github.julien-truffaut" %% "monocle-core"                      % "2.0.0",
+    "com.github.julien-truffaut" %% "monocle-macro"                     % "2.0.0",
+    compilerPlugin("com.github.ghik" % "silencer-plugin"    % "1.7.1" cross CrossVersion.full),
```

Comment:
minor : `1.7.5`

Reply:
Somehow, CGT FE is not accepting scala 2.12.13 so cannot use 1.7.5 version here.
Will double check it